core/src/com/unciv/logic/map/mapunit/movement/UnitMovement.kt

```diff
@@ -602,7 +602,13 @@ class UnitMovement(val unit: MapUnit) {
     private fun canAirUnitMoveTo(tile: Tile, unit: MapUnit): Boolean {
         // landing in the city
         if (tile.isCityCenter()) {
-            if (tile.airUnits.filter { !it.isTransported }.size < 6 && tile.getCity()?.civ == unit.civ)
+            val airUnitCapacity = tile.getCity()?.civ!!.gameInfo.ruleset.modOptions.constants.airUnitCapacity
```

Comment:
Let's put this logic in one single place, probably on tile

Also, you can't be sure this is a city, so !! WILL throw an exception on aircraft carriers

Reply:
> Let's put this logic in one single place, probably on tile
Done 👍

> Also, you can't be sure this is a city, so !! WILL throw an exception on aircraft carriers
I have tested it, and the carrier doesn't crash the game. I think the reason is because of the if statement if (tile.isCityCenter) which ensures that the tile is a city center and not a carrier."